core-bundle/contao/languages/en/tl_content.xlf

```diff
@@ -731,6 +731,18 @@
       <trans-unit id="tl_content.table.1">
         <source>Import table items from a CSV file</source>
       </trans-unit>
+      <trans-unit id="tl_content.dl_label.0">
+        <source>Description list content</source>
+      </trans-unit>
+      <trans-unit id="tl_content.dl_label.1">
+        <source>Enter a list of terms and details for the description list. You can conditionally leave the term field empty to create multiple details for one term.</source>
```

Comment:
What does **conditionally** mean in this context? Under what conditions can one leave the term field empty?

Reply:
that's how descriptions lists work. You can have multiple descriptions per term.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#single_term_multiple_descriptions

Reply:
Then it should just be _You can leave the term field empty to create multiple details for one term_.